samples/net/secure_mqtt_sensor_actuator/overlay_static_insecure.conf

```diff
@@ -0,0 +1,18 @@
+# Config to disable TLS and DHCPv4, allowing insecure MQTT and a static IP address.
+# This allows quick testing of the application without need for a DHCP server or secure MQTT broker set up.
+# Only recommended for quick sampling/testing.
+# Usage: west build -b <board> -- -DOVERLAY_CONFIG=overlay_static_insecure.conf
+
+# Disable MQTT with TLS
+CONFIG_MQTT_LIB_TLS=n
+
+# Disable DHCP
+CONFIG_NET_DHCPV4=n
+
+# Insecure MQTT uses port 1883
+CONFIG_MQTT_BROKER_PORT=1883
+
+# Static IP address config
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="169.254.0.2"
+CONFIG_MQTT_BROKER_IPV4_ADDR="169.254.0.1"
```

Comment:
Please use the IP address space we use in other samples (the addresses are reserved for documentation use).
So here
```
CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1"
CONFIG_NET_SAMPLE_MQTT_BROKER_IPV4_ADDR="192.0.2.2"
```

Reply:
Done.